buildspecs/aix_ppc-64.spec

```diff
@@ -221,6 +221,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="module_zlib" value="true"/>
 		<flag id="opt_annotations" value="true"/>
 		<flag id="opt_bigInteger" value="true"/>
+		<flag id="opt_criuSupport" value="false"/>
```

Comment:
New flags default to false, it's not necessary to change all these spec files to add the false value. It's not harmful, so we can add them if you really think it's necessary.

Reply:
I dont think its necesarry